nikola/plugins/command/plugin.py

```diff
@@ -179,9 +179,18 @@ def list_installed(self):
         plugins.sort()
         print('Installed Plugins:')
         print('------------------')
+        maxlength = max(len(i[0]) for i in plugins)
+        if self.site.colorful:
+            formatstring = '\x1b[1m{0:<{2}}\x1b[0m  at {1}'
```

Comment:
On Windows, we just disable the colors everywhere. But Windows 10 *does* support ANSI escape codes, so we can enable colors if the user is on 10.